### DISCUSSION TITLE
Excluding files from wheels

### DISCUSSION THREAD

**Discussion by NiceneNerd** [PyO3/maturin#694] [Q&A] [answered]
I am using maturin as a convenient way to shift some heavy processing in a Python app to Rust (so, mixed Python/Rust setup), which is going well except for building the wheels. My project has a large number of UI-related files which are part of the source folders but are normally excluded from sdist and wheel builds. However, when I use `maturin build` all of these files are included, massively bloating the resulting wheel. Is there some way I'm missing that I can use to exclude these files?

**Comment by messense** [comment, marked as answer]
See https://github.com/PyO3/maturin/issues/554#issuecomment-851352109

Edit: #554 is only for sdist, so it looks like we don't have a way to exclude files for wheels yet.

**Reply by messense** [reply]
Will https://github.com/PyO3/maturin/pull/695 work for you? It'll exclude files according to `.gitignore`.

**Reply by NiceneNerd** [reply]
Ooh, nice, looks good to me

**Reply by messense** [reply]
Please give v0.12.2-beta.1 a try, you can install it from PyPI with `pip install -U --pre maturin`.

**Reply by NiceneNerd** [reply]
Works perfectly, thanks
